bundles/org.openhab.binding.smhi/src/main/java/org/openhab/binding/smhi/internal/Forecast.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.smhi.internal;
+
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * A class containing a forecast for a specific point in time.
+ *
+ * @author Anders Alfredsson - Initial contribution
+ */
+@NonNullByDefault
+public class Forecast implements Comparable<Forecast> {
+    protected final ZonedDateTime validTime;
```

Comment:
Any reason this is protected rather than private?

Reply:
I (apparently mistakenly) thought protected was to enable instances to see each other's fields (in the compareTo method), but seems I got it mixed up with Ruby's access modifiers.